docs/source/first-steps.rst

```diff
@@ -0,0 +1,32 @@
+First Steps
+===========
+
+Work Laptop
+-----------
+Check your personal email for login instructions from IT.
+
+Work email
+----------
+Look for an email asking you to change your password that mentions “the Center
+for Reimagining Learning”. Submit a new password, and sign into your now
+accessible account.
+
+Jira/Confluence
+---------------
+Set up an account on Jira/Confluence via Google using your work email address.
```

Comment:
Make `Jira/Confluence` a link to the page where you should go to register a new account.

Reply:
Also should you log in with SSO or with a uname/pwd? (pretty sure it's SSO)

Reply:
The [link to sign up](https://id.atlassian.com/signup?continue=https%3A%2F%2Fopenedx.atlassian.net%2Flogin%3FredirectCount%3D1%26dest-url%3Dhttps%253A%252F%252Fopenedx.atlassian.net%252Fjira%252Fprojects%26application%3Djira&application=jira) is very long compared to the [link to the main page](https://openedx.atlassian.net/jira/projects), so I was thinking of using the second link and adding instructions on where to sign in.

Reply:
I've added the links